cloud/amazon/rds.py

```diff
@@ -42,7 +42,7 @@
       - The type of database.  Used only when command=create.
```

Comment:
Please add a little note in the docs e.g.
~~~
- mariadb was added in version 2.2
~~~